README.md

```diff
@@ -42,10 +42,81 @@ Functions
     with summary statistics for a list of columns of a few players or
     teams.
 
+Preparation Usage for Scraping with `nba_scraper()`
+---------------------------------------------------
+
+The `rsketball::nba_scraper` is based on Selenium (or specifically
+RSelenium) which enables automated web browsing through “drivers”. To
+use it, please ensure that `Docker` is installed. For installation
+instructions, please follow the [guide to Docker installation based on
+your OS
+type](https://ubc-mds.github.io/resources_pages/installation_instructions/).
+Docker will be used to pull relevant images that when executed as
+containers, will serve as the “drivers” for Selenium.
+
+The following steps are required only for the `nba_scraper` function. If
+you already have the scraped data file and wish to use the other
+functions (`nba_boxplot`, `nba_rank`, `nbastats`), there is no need to
+proceed with these steps.
+
+**Step 1 (Command line/Terminal): Preparation Step (Docker container)**
+
+Pull docker image with the following code in Terminal. We will stick to
+Chrome since it seems compatible with Windows while Firefox is not.
+
+    docker pull selenium/standalone-chrome
+
+**Critical step about setting ports and memory allocation:**
+
+We need to set up the Docker container default port 4444 to our computer
+host port 4445. Keep this port number as inputs for the `nba_scraper`
+function. We will also allocate the virtual memory of the container to
+2Gb for it to scrape effectively.
+
+Run the following code in Terminal:
+
+    docker run -d -p 4445:4444 --shm-size 2g selenium/standalone-chrome
+
+Verify that the docker container is in operation with following code in
+Terminal:
+
+    docker ps 
+
+**Step 2 (R/RStudio): Scraping with `nba_scraper`**
+
+Now that the container is running with the allocated memory and assigned
+port, we can proceed with testing
+
+    library(rsketball)
+    # Scrape regular season 2018/19 using "chrome" driver and
+    # create "nba_2018_regular" tibble in Global Env
+    nba_scraper(season_year = 2018, season_type = "regular", 
+                csv_path = "nba_2018_regular.csv",
+                port=4445L, # Insert the port number with a L suffix as shown
+                sel_browser = "chrome",
+                nba_data_env_name = "nba_2018_regular")
```

Comment:
Remember removing the environment parameter from the documentation

Reply:
this has been changed accordingly with a more recent commit!